modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm

```diff
@@ -0,0 +1,142 @@
+/datum/species/hemophage
+	name = "Hemophage"
+	id = SPECIES_HEMOPHAGE
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,DRINKSBLOOD, HAS_FLESH, HAS_BONE)
+	inherent_traits = list(
+		TRAIT_ADVANCEDTOOLUSER,
+		TRAIT_CAN_STRIP,
+		TRAIT_NOHUNGER,
+		TRAIT_NOBREATH,
+		TRAIT_VIRUSIMMUNE,
+	)
+	inherent_biotypes = MOB_UNDEAD|MOB_HUMANOID
+	mutant_bodyparts = list("wings" = "None")
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | ERT_SPAWN
+	exotic_bloodtype = "U"
+	use_skintones = TRUE
+	mutantheart = /obj/item/organ/heart/vampire
+	mutanttongue = /obj/item/organ/tongue/vampire
+	limbs_id = SPECIES_HUMAN
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+	var/info_text = "You are a <span class='danger'>Hemophage</span>. You will slowly but constantly lose blood if outside of a coffin. If inside a coffin, you will slowly heal. You may gain more blood by grabbing a live victim and using your drain ability."
```

Comment:
```suggestion
	/// Some starter text sent to the vampire initially, because vampires have shit to do to stay alive.
	var/info_text = "You are a [span_danger("Hemophage")]. You will slowly but constantly lose blood if outside of a coffin. If inside a coffin, you will slowly heal. You may gain more blood by grabbing a live victim and using your drain ability."
```

Reply:
I had made changes to the info_text too, I don't know if you saw it or not.

Reply:
I did, it however, really did not like said changes,

Reply:
Ah. Oh well.